subsys/logging/log_minimal.c

```diff
@@ -12,7 +12,7 @@
 
 static void minimal_hexdump_line_print(const char *data, size_t length)
 {
-	for (int i = 0; i < HEXDUMP_BYTES_IN_LINE; i++) {
+	for (unsigned int i = 0; i < HEXDUMP_BYTES_IN_LINE; i++) {
 		if (i < length) {
```

Comment:
I would use `size_t` here instead of `unsigned int`. In 64 bits, `unsigned int` is 4 bytes and `size_t` is 8 bytes and as you are comparing them ... using size_t is the right type to use.

Reply:
yes agreed, always use `size_t` when a size in bytes is expressed

Reply:
Addressed. Please review again. Thanks